aws-event-sources-operator/helm-charts/aws-event-sources/crds/crd-awscodecommit.yaml

```diff
@@ -0,0 +1,182 @@
+# Copyright (c) 2020 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: awscodecommitsources.sources.triggermesh.io
```

Comment:
Creating duplicates of everything will be a management hell. We should quickly come up with a better release process (don't have to be part of that PR).

Reply:
there are some tools used to manage the operator code, `operator-sdk` is the main one and `operator-courier` is another. most of the files are generated using these tools, ideally generating a new version of the operator would involve manually updating files in the `deploy/` directory and then generating the rest using the tools.